src/PanelTraits/AutoSet.php

```diff
@@ -45,24 +47,25 @@ public function setFromDb()
                     'autoset' => true,
                 ]);
             }
-        }, $this->getDbColumnsNames());
+        }, $this->getDbColumnsNames($filtered));
     }
 
     /**
      * Get all columns from the database for that table.
      *
-     * @return [array]
+     * @return array
      */
     public function getDbColumnTypes()
     {
-        $table = $this->model->getTable();
-        $conn = $this->model->getConnection();
-        $table_columns = $conn->getDoctrineSchemaManager()->listTableColumns($table);
-
-        foreach ($table_columns as $key => $column) {
-            $column_type = $column->getType()->getName();
-            $this->db_column_types[$column->getName()]['type'] = trim(preg_replace('/\(\d+\)(.*)/i', '', $column_type));
-            $this->db_column_types[$column->getName()]['default'] = $column->getDefault();
+        if (empty($this->db_column_types)) {
+            $table_columns = $this->getDbColumns();
+
+            foreach ($table_columns as $key => $column) {
+                $column_type = $column->getType()->getName();
```

Comment:
There is a field called `$column_type` (**type**) where it is assigned something from a getter called `getName()` (**name**). Huh?